test/functional/test_framework/test_framework.py

```diff
@@ -323,12 +323,13 @@ def shutdown(self):
 
         self.log.debug('Closing down network thread')
         self.network_thread.close()
-        if self.success == TestStatus.FAILED:
-            self.log.info("Not stopping nodes as test failed. The dangling processes will be cleaned up later.")
-        else:
-            self.log.info("Stopping nodes")
-            if self.nodes:
-                self.stop_nodes()
+        if self.nodes:
```

Comment:
In commit "qa: Only log node stopping info if we have any" (9d0e6a1b2aa5be186550658ff85e5ad643554338)

Would be helpful if commit description stated motivation for this change. After looking at the `stop_nodes()` method implementation it seem like this is a logging-only change and point is just to remove stopping log statements when there are no nodes. But it would be good if actual intent of change were explicit.